src/engine/database/gremlin.rs

```diff
@@ -183,10 +185,17 @@ impl GremlinEndpoint {
         Ok(GremlinEndpoint {
             host: env_string("WG_GREMLIN_HOST")?,
             port: env_u16("WG_GREMLIN_PORT")?,
-            user: env_string("WG_GREMLIN_USER")?,
-            pass: env_string("WG_GREMLIN_PASS")?,
+            user: match env_string("WG_GREMLIN_USER") {
+                Ok(v) => Some(v),
+                Err(_) => None
+            },
+            pass: match env_string("WG_GREMLIN_PASS") {
+                Ok(v) => Some(v),
+                Err(_) => None
```

Comment:
Same as the comment on `user` above. Probably easier to call `var_os` and `map` directly than convert an `Option` to a `Result` and back again.